### PR TITLE
[CLEANUP] Consistently use one space after a colon in a rule in add-ons.

### PR DESCRIPTION
**Pull request by oliverklee**
This fixes the following scss-lint warning:

"Colon after property should be followed by 1 space"

This is part of an effort to fix all scss-lint warnings in yaml.

### CODE REVIEW

**Comment by oliverklee** [comment]
Generally, I agree on having only one property per line. However, for this pull request, I'd like to limit the changes to the "one space after the colon" issue to make it easier to review. I'll planning fix the other issues in separate pull requests.

File: sass/yaml-sass/mixins/_yaml-mixins-core.scss [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 // BOX SIZING
 @mixin ym-box-sizing($type: border-box) {
-	-webkit-box-sizing:	$type;
-	-moz-box-sizing:	$type;
-	box-sizing:			$type;
+	-webkit-box-sizing: $type;
+	-moz-box-sizing: $type;
+	box-sizing: $type;
```
Comment (michsch) [inline sass/yaml-sass/mixins/_yaml-mixins-core.scss:9]: I prefer to indent the vendor prefixes:

``` css
@mixin ym-box-sizing($type: border-box) {
  -webkit-box-sizing: $type;
     -moz-box-sizing: $type;
          box-sizing: $type;
}
```

Comment (oliverklee) [inline sass/yaml-sass/mixins/_yaml-mixins-core.scss:9]: Indenting the vendor prefixes will trigger the warning in scss-lint. scss-lint expects exactly one space after the colon. in a rule.

File: sass/yaml-sass/navigation/_hlist.scss [hunk 2]
```diff
@@ -5,53 +5,53 @@
 	.ym-hlist {
 		/* (en) containing floats in IE */
 		/* (de) Einfassen der Floats im IE */
-		width:100%;
-		overflow:hidden;
-		position:relative;	//(en|de) Bugfix:IE - collapsing horizontal margins
-		line-height:1em;
+		width: 100%;
+		overflow: hidden;
+		position: relative;	//(en|de) Bugfix:IE - collapsing horizontal margins
```
Comment (oliverklee) [inline sass/yaml-sass/navigation/_hlist.scss:10]: The comments actually should go in a separate line. However, that should go into a separate pull request.

File: sass/yaml-sass/mixins/_yaml-mixins-core.scss (hunk 1 above)
Comment (michsch) [inline sass/yaml-sass/mixins/_yaml-mixins-core.scss:9]: And what's about the spaces at the beginning of the line?

File: sass/yaml-sass/navigation/_hlist.scss (hunk 2 above)
Comment (michsch) [inline sass/yaml-sass/navigation/_hlist.scss:10]: Agreed

**Comment by michsch** [comment]
**Reviewed**

In my opinion this is a really good pull request! I thought about this issues multiple times.
I just have a few additional points we should edit in separate pull requests:
- one CSS property per line
- use Sass Mixins instead of vendor prefixes multiple times

@djesse Ready to merge.

File: sass/yaml-sass/mixins/_yaml-mixins-core.scss (hunk 1 above)
Comment (oliverklee) [inline sass/yaml-sass/mixins/_yaml-mixins-core.scss:9]: That something for a separate pull request as well.
